app/views/hub/dashboard/_returns_by_status_content.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2><%= I18n.t("hub.dashboard.show.returns_by_status") %></h2>
+<div class="counts">
+  <% unless @returns_by_status_stage.present? %>
+    <div>
+      <span class="count"><%= @returns_by_status_count %></span>
+    </div>
+    <div class="description"><%= I18n.t("hub.dashboard.show.total_number_of_returns") %></div>
+  <% else %>
+    <div>
+      <span class="count"><%= @returns_by_status_count %></span>
+      <span>/<%= @returns_by_status_total %></span>
+    </div>
+    <div class="description"><%= I18n.t("hub.dashboard.show.total_returns_in", stage: "<span class=\"\"style='font-weight:bold'>#{I18n.t("hub.tax_returns.stage.#{params[:stage]}")}</span>".html_safe).html_safe %></div>
```

Comment:
could this html move into the strings file? hub.clients.bulk_action.count_html looks like a precedent, and i think it would be easier to read. also, there are a lot of places we use `<strong>` in the strings file instead of `font-weight:bold`, so that might be a convention worth adopting here too

Reply:
Converted to `strong` and moved into the strings file.